Application/Extensions/StringExtensions.cs

```diff
@@ -11,9 +11,17 @@ namespace Application.Extensions;
 public static class StringExtensions
 {
     [return: NotNullIfNotNull("value")]
-    public static string? ToSeoUrl(this string? value)
+    public static string? ToSeoUrl(this string? value, bool replaceForwardSlash = true)
     {
-        var seo = value?
+        if (string.IsNullOrEmpty(value))
+            return value;
+
+        var seo = value;
+
+        if (replaceForwardSlash)
+            seo = seo.Replace("/", "");
```

Comment:
Are we expecting forward slash for the Seo Url?

Reply:
If not we can incorporate the forward slash replacing logic with the code below.

Reply:
I had originally put it within the `UrlUnsafeCharacters` - but on certain occasions we do need to leave the forward slashes, if part of full path.  See `ToSeoUrl_ReturnsKebabCase_WhenDirectory` and `SeoRouteConvention` within solution,

Reply:
I see what you mean it's also used with directory, make sense.